extensions/chocolatey-vscode.extension/extensions/Install-VsCodeExtension.ps1

```diff
@@ -38,16 +42,36 @@ function Install-VsCodeExtension {
         [string]$extensionId
     )
 
-    Write-Verbose "Locating Visual Studio Code installation directory..."
-    $codePath = Get-AppInstallLocation "Microsoft Visual Studio Code" | Where-Object { Test-Path "$_\bin\code.cmd" } | Select-Object -first 1 | ForEach-Object { "$_\bin\code.cmd" }
-   
-    if (!$codePath) {
-      Write-Error "Visual Studio Code installation directory was not found."
-      throw "Visual Studio Code installation directory was not found."
+    function InstallExtension($installLocation, $executablePath) {
+        if (!$installLocation) {
+            return
+        }
+
+        Write-Verbose "Visual Studio Code installation found at $installLocation"
+    
+        Write-Host "Installing Visual Studio Code extension $extensionId in $installLocation..."
+        Start-ChocolateyProcessAsAdmin -ExeToRun $installLocation -Statements "--install-extension",$extensionId    
```

Comment:
Do installing extensions require administrative privileges?

If not, we could set `-Elevated:$false` as an arguments.
This would allow users not running choco in elevated mode to install the extension in non-admin mode.